tox.ini

```diff
@@ -10,49 +10,54 @@ isolated_build = True
 [testenv]
 commands = python -m pip install --upgrade pip
 
-[testenv:dev]
-description = development environment
-usedevelop = True
-deps =
-    black
-    pip-tools
-    flake8
-    flakehell
-    pytest
-    pytest-datafiles
-commands =
-    python -m pip install --upgrade pip
-    pip-compile requirements/base.in
-    pip-compile requirements/dev.in
-    pip-compile requirements/docs.in
-    pip-compile requirements/tests.in
-    pip install -q -e . -r requirements/dev.txt
 
 [testenv:tests]
 deps =
-    -rrequirements/base.txt
+    -rrequirements.txt
     pytest
     pytest-datafiles
 commands = python -m pytest --basetemp={envtmpdir} {posargs}
 
+
 [testenv:linting]
-skip_install = true
 deps =
-    -rrequirements/base.txt
+    -rrequirements.txt
     pylint
 commands = pylint src/bo4e
 
+
 [testenv:coverage]
 deps =
     {[testenv:tests]deps}
     coverage
 commands =
     coverage run -m pytest --basetemp={envtmpdir} {posargs}
+    coverage html --omit .tox/*,unittests/*
     coverage report --fail-under 80 --omit .tox/*,unittests/*
 
 
+[testenv:dev]
+usedevelop = True
+deps =
+    {[testenv:tests]deps}
+    {[testenv:linting]deps}
+    {[testenv:coverage]deps}
+    black
```

Comment:
black wird in den requirements installiert

Reply:
Ach, das meintest du mit requirements.txt ^^

Reply:
Habs mal mit einem Kommentar gepinnt, sonst hat man im Zweifelsfall das gleiche Problem wieder, wenn jemand es dort anpasst:
ebf92f62e8d96fc70845f267ff25a9648f09db48